server/server.go

```diff
@@ -0,0 +1,131 @@
+package server
```

Comment:
All this code should be part of the library, not CoreDHCP. Would you mind creating a PR for https://github.com/insomniacslk/dhcp to modify https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/server4/server.go?

It will make ti more consistent with the v6 one: https://github.com/insomniacslk/dhcp/blob/master/dhcpv6/server6/server.go